python/py-spopt/Portfile

```diff
@@ -60,6 +59,17 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-spaghetti \
                     port:py${python.version}-tqdm
 
+    if {${python.version} == 39} {
+        python.pep517   no
+        version         0.5.0
+        revision        0
+        checksums       rmd160  4d387eb900763e5e49b41b1f68ace0c0aa3a029e \
+                        sha256  c3d6572c1167b5d59a98781f8697cc6ba10f1fbf65ecf1ca1bad1aaa9d4ce511 \
+                        size    29182492
+
+        depends_build-append port:py${python.version}-setuptools
+    }
+
     test.run        yes
     test.cmd        py.test-${python.branch}
     test.target
```

Comment:
the `test.cmd` and `test.target` are set to the same values as is done by default through the `python` PG - where `test.run` uses as default `python.test_framework pytest`.